scripts/migration/lib.sh

```diff
@@ -18,6 +18,7 @@ SC_CONFIG_FILES=(
   "defaults/sc-config.yaml"
   "common-config.yaml"
   "sc-config.yaml"
+  "secrets.yaml"
```

Comment:
What changed to require this change? Looks unrelated to this PR but I could be wrong.

Reply:
Gotcha! In the future I would suggest to send such a separate fix as a separate PR to make it clear since GitHub doesn't have a good "commit-by-commit" review system but instead makes you review all the changes at once. Not a blocker here though. Just don't squash it with the rest of the commits. :)